### PR TITLE
Project migration to webpack 5

### LINKED ISSUE #56: Migrate to webpack 5
## Provide a description of the task

Scope: ckeditor/ckeditor5#9799

It is already estimated in ckeditor/ckeditor5#10668.

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Other (tools): Project migration to webpack 5. Closes #56.

Tests (tools): Added a few tests that mostly cover loaders in webpack.

### CODE REVIEW

**Comment by pomek** [comment]
Please, do not merge these changes. It needs to be synchronized with CKEditor 5.

**Comment by pomek** [comment]
Ready to review.

image

File: .travis.yml [hunk 1]
```diff
@@ -14,5 +14,5 @@ before_install:
 install:
   - yarn install
 script:
-  - yarn run test
+#  - yarn run test
```
Comment (pomek) [inline .travis.yml:17]:
```suggestion
  - yarn run test
```

**Review by przemyslaw-zan — APPROVED** [review]
After today's live review meeting, LGTM

**Pull request merged**